omise/classes/charge.php

```diff
@@ -0,0 +1,64 @@
+<?php
```

Comment:
@nimid Checking. But thought nowhere to comment more. Going to finalise soon-ish